packages/nix/polymc/default.nix

```
@@ -76,6 +71,7 @@ mkDerivation rec {
     # xorg.xrandr needed for LWJGL [2.9.2, 3) https://github.com/LWJGL/lwjgl/issues/128
     wrapProgram $out/bin/polymc \
       "''${qtWrapperArgs[@]}" \
+      --set POLYMC_JAVA_PATHS ${jdk}/lib/openjdk/bin/java:${jdk8}/lib/openjdk/bin/java \
```

This should be done with `--prefix`. Otherwise users can't declare custom paths.

```suggestion
      --prefix POLYMC_JAVA_PATHS : ${jdk}/lib/openjdk/bin/java:${jdk8}/lib/openjdk/bin/java \
```